sinks/multi_sink.py

```diff
@@ -0,0 +1,29 @@
+import subprocess
+import sys
+
+MAX_READLINES = 1000
+
+
+class Sinks:
+
+    def __init__(self, sink_commands):
+        self.sinks = []
+        self.lines_sent = 0
+        for param in sink_commands:
+            print(f'Executing: {param}')
+            self.sinks.append(subprocess.Popen(param.split(), stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.STDOUT))
```

Comment:
I think that maybe we should protect this code if the Popen fails to create the process. I think is acceptable that one of the sink dies at some point making both sinks to fail, but we should at least be able to ignore the sinks that fail to be launched

Reply:
Popen does not fail if the command can not be executed (tried with wrong sink name and with a sink raising exception) but error was not visible. I added code to show sink output if any. Now the output is like this:

Executing 1: python sink_a.py
Executing 2: python sink_b.py
Error in 1: b'Traceback (most recent call last):\n  File "sink_a.py", line 3, in <module>\n    raise Exception()\nException\n'
10 lines sent